src/main/java/no/digipost/api/client/util/DigipostApiMock.java

```diff
@@ -164,13 +178,17 @@ public void handleRequest(HttpServerExchange httpContext) throws Exception {
         } else if(requestPath.equals("/getdocumentstatus")) {
             CloseableHttpResponse response = requestsAndResponsesMap.get(Method.GET_DOCUMENT_STATUS).getResponse(httpContext.getRequestPath());
             httpResponse = response.getStatusLine().getStatusCode();
-            response.getEntity().writeTo(bao);
-
+            HttpEntity entity = response.getEntity();
```

Comment:
At du ikke har endra dette til `Optional.ofNullable(response.getEntity()).map(e -> e.writeTo(bao))` for å få dette på en one liner er for meg fullstendig uforståelig

Reply:
🤡

Reply:
:trollface:

Reply:
🙄

Reply:
Prøv a.

Reply:
haha, crap, erre no' exceptions inne i bildet, mon tro 🤔

Reply:
:synkendeskip: